tests/Aspire.EndToEnd.Tests/.runsettings

```diff
@@ -1,5 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <RunSettings>
+  <xunit>
```

Comment:
Is this actually working for you? My understanding was that these are case sensitive (because thanks XML), and these _should_ be `xUnit` (with the capital U). https://xunit.net/docs/runsettings

Reply:
![image](https://github.com/user-attachments/assets/27827217-c56a-41e3-b0a4-49b263bcb156)

Reply:
Ah, thank you. I will fix that.